blog/2015-03-05-JBDS-8.1.0.Beta1-for-luna.adoc

```diff
@@ -0,0 +1,92 @@
+= Beta Time for JBoss Developer Studio 8.1.0 and JBoss Tools 4.2.3 
+:page-layout: blog
+:page-author: akazakov
+:page-tags: [release, jbosstools, devstudio, jbosscentral]
+
+We are ready with link:/downloads/devstudio/luna/8.1.0.Beta1.html[Red Hat JBoss Developer Studio 8.1.0 Beta1] and link:/downloads/jbosstools/luna/4.2.3.Beta1.html[JBoss Tools 4.2.3 Beta1] for Eclipse Luna SR2.
+
+image::images/jbosstools-jbdevstudio-blog-header.png[]
+
+== Updating from JBoss Tools 4.2.x or JBoss Developer Studio 8.0.x
+
+If you have JBoss Tools 4.2.x or JBoss Developer Studio 8.0.x already installed, just run:
+
+    Help > Check for updates
```

Comment:
this wont work for beta's. 

here you need to use the development updatesite explicitly.